### PR TITLE
Upload of log transformed data points.

### PR DESCRIPTION
**Pull request by forus**
DATA_TYPE=L

### CODE REVIEW

File: src/main/groovy/org/transmartproject/batch/highdim/beans/AbstractTypicalHdDataStepsConfig.groovy [hunk 1]
```diff
@@ -93,11 +93,20 @@ abstract class AbstractTypicalHdDataStepsConfig implements StepBuildingConfigura
...
+        if (dataType == 'L') {
+            processors << calculateRawValueFromTheLogItemProcessor()
+        }
         compositeOf(*processors)
     }
 
     @Bean
     @StepScope
+    CalculateRawValueFromTheLogItemProcessor calculateRawValueFromTheLogItemProcessor() {
+        new CalculateRawValueFromTheLogItemProcessor()
```
Comment (cataphract) [inline src/main/groovy/org/transmartproject/batch/highdim/beans/AbstractTypicalHdDataStepsConfig.groovy]: Seems like job scope is enough.

File: src/main/groovy/org/transmartproject/batch/highdim/platform/AbstractPlatformJobSpecification.groovy [hunk 2]
```diff
@@ -37,18 +37,16 @@ abstract class AbstractPlatformJobSpecification
...
     void validate(ExternalJobParametersInternalInterface ejp)
             throws InvalidParametersFileException {
-        ejp.with {
-            mandatory PLATFORM
-            mandatory TITLE
-            mandatory ANNOTATIONS_FILE
-        }
+            mandatory ejp, PLATFORM
+            mandatory ejp, TITLE
+            mandatory ejp, ANNOTATIONS_FILE
```
Comment (cataphract) [inline src/main/groovy/org/transmartproject/batch/highdim/platform/AbstractPlatformJobSpecification.groovy]: ws

**Comment by jochemb** [comment]
Could you add to the docs what happens when a users sets `DATA_TYPE=L`

**Comment by forus** [comment]
@jochemb Where should I add this kind of information? Should it be a new document?

**Comment by jochemb** [comment]
@forus https://github.com/thehyve/transmart-batch/blob/master/docs/hd-data-processing-details.md would be the place

**Pull request merged**